.github/workflows/create-release.yml

```diff
@@ -39,7 +45,7 @@ jobs:
       run: |
         source python/etc/profile.d/conda.sh
         conda activate venv_sct
-        pip freeze | grep -v "-e git+" | grep -v "torch" | grep -v "certifi @ file" > requirements-freeze.txt
+        pip freeze | grep -v "-e git+" | grep -v "certifi @ file" > requirements-freeze.txt
```

Comment:
Ah shoot, I guess `grep -v torch` is necessary after all: https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/13058658736/job/36436133378?pr=4779#step:5:840

I guess we can go ahead and adopt the old approach we used to have, namely copying `torch<2.6` into the frozen requirements (but still excluding it from the `pip freeze` output).